scripts/tasks/lint-imports.js

```diff
@@ -14,6 +14,46 @@ module.exports = function() {
   const rushJsonPath = findConfig('rush.json');
   const rootFolder = path.dirname(rushJsonPath);
   const rush = readConfig(rushJsonPath);
+
+  const allowRelativeImportExamples = [
+    // These were added to reduce the initial ramifications of disabling relative imports across all examples,
```

Comment:
No, scss files are excluded by the filter. This comment is regarding [hundreds of line chunky data helpers](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/examples/PeoplePickerExampleData.ts) that are not just used by examples, but also codepen-loader and tests. They'd have to be put first in example files which would obfuscate the actual example. @kenotron has some ideas for folding that content so it can be included in examples with obfuscating the actual example.

Reply:
Just to be clear, the codepen-loader references the example data helpers so it can include their contents inline for examples which import those helpers.